controllers/admin/AdminShoprunbackController.php

```diff
@@ -169,10 +174,20 @@ public function initContent()
             'srbtoken'                          => RestClient::getClient()->getToken(),
             'link'                              => $link,
             'template'                          => $template,
-            'admin_module_ajax_url_shoprunback' => 'index.php?controller=AdminShoprunback&token=' . Tools::getAdminTokenLite('AdminShoprunback')
+            'admin_module_ajax_url_shoprunback' => 'index.php?controller=AdminShoprunback&token=' . Tools::getAdminTokenLite('AdminShoprunback'),
+            'enable_return_btn' => (Configuration::get('SRB_ENABLE_RETURN_BTN') !== false) ? Configuration::get('SRB_ENABLE_RETURN_BTN') : '0'
```

Comment:
does Configuration::get('SRB_ENABLE_RETURN_BTN') always got a value (boolean).

If yes, reverse the condition and remove the !== 

`
Configuration::get('SRB_ENABLE_RETURN_BTN')  ?  '0' : Configuration::get('SRB_ENABLE_RETURN_BTN')
`

Reply:
that configuration would mean that if it has a value, we take 0, isn't it ? 
Configuration::get('SRB_ENABLE_RETURN_BTN') ? Configuration::get('SRB_ENABLE_RETURN_BTN') : '0' would be better ? I am doubting all of a sudden !

Reply:
you make me doubting now :p 

If after testing, it works, I'm ok with that